pages/facts.html

```diff
@@ -4,9 +4,94 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/stylesheets/style.css" rel="stylesheet">
+  <link href="/stylesheets/facts.css" rel="stylesheet">
+  <title>Fun Facts!</title>
 </head>
 <body>
-  
+  <header>
+    <h1>Community & Human Services in King County</h1>
+    <nav class="nav">
+    <a href="index.html">Home</a>   <a href="gallery.html">Gallery</a>
+    </nav>
```

Comment:
Nitpick: spacing/indentation

```suggestion
    <nav class="nav">
        <a href="index.html">Home</a>
        <a href="gallery.html">Gallery</a>
    </nav>
```